.github/workflows/ci.yml

```diff
@@ -0,0 +1,65 @@
+
+# See https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+
+  Package:
+    runs-on: ubuntu-latest
+    container: registry.opensuse.org/yast/head/containers/yast-ruby:latest
+
+    steps:
+
+    - name: Git Checkout
+      uses: actions/checkout@v1
+
+    - name: Install Dependencies
+      run: |
+        zypper --non-interactive install --no-recommends \
+          autoyast2 \
+          libxml2-tools \
+          jing \
+          trang \
+          yast2 \
+          yast2-add-on \
+          yast2-audit-laf \
+          yast2-auth-client \
+          yast2-auth-server \
+          yast2-bootloader \
+          yast2-country \
+          yast2-dhcp-server \
+          yast2-dns-server \
+          yast2-firewall \
+          yast2-firstboot \
+          yast2-ftp-server \
+          yast2-geo-cluster \
+          yast2-http-server \
+          yast2-installation \
+          yast2-iscsi-client \
+          yast2-kdump \
+          yast2-mail \
+          yast2-network \
+          yast2-nfs-client \
+          yast2-nfs-server \
+          yast2-nis-client \
+          yast2-nis-server \
+          yast2-ntp-client \
+          yast2-online-update-configuration \
+          yast2-printer \
+          yast2-proxy \
+          yast2-registration \
+          yast2-samba-client \
+          yast2-samba-server \
+          yast2-sound \
+          yast2-squid \
+          yast2-tftp-server
```

Comment:
this list looks quite hard to maintain. Maybe it will be easier to install src package for yast2-schema which should itself install all build requires?